### ISSUE TITLE
Public key file option documentation broken

### ISSUE THREAD

**Issue by adamjrat** [containers/skopeo#2043]
The documentation page says:
**--public-key-file public key file
File containing the public keys to use when verifying signatures. If this is not specified, keys from the GPG homedir are used.**

Skopeo says no!
```
$ skopeo standalone-verify --public-key-file
FATA[0000] unknown flag: --public-key-file
$ rpm -qi skopeo
Name        : skopeo
Epoch       : 2
Version     : 1.9.4
...

```

It is without the double hyphen like this (sorry not a working example): 
```
$ skopeo standalone-verify public-key-file
FATA[0000] Usage: skopeo standalone-verify manifest docker-reference key-fingerprint signature
```
ref comments: (show resolved) if interested,
https://github.com/containers/skopeo/pull/1950/files#discussion_r1152658274
 
further: man page says nothing about this!?

Love your work, and this seems trivial - until it drives you nuts wondering why it doesn't work!

**Comment by edsantiago** [comment]
Where are you finding this "documentation page"? Is it possible that you're reading online documentation? If so, that's your problem: the only valid documentation for the version you have installed is the man page on that system.

In any case, you seem to be running `1.9.4`, and the `--public-key-file` option was added in the PR you linked to, which was then released in `1.12`.

**Comment by mtrmac** [comment]
Assuming that this is indeed explained by running an older version of Skopeo, please reopen if that’s not the case.
